kbs/docker/Dockerfile.rhel-ubi

```diff
@@ -0,0 +1,28 @@
+# Use Fedora to build.
+FROM registry.fedoraproject.org/fedora-minimal:40 as builder
+
+# Install build dependencies from Fedora repos.
+RUN microdnf -y --setopt=install_weak_deps=0 install \
+cargo pkg-config perl-FindBin openssl-devel perl-lib perl-IPC-Cmd perl-File-Compare perl-File-Copy tpm2-tss-devel clang-devel protobuf-compiler \
+tar gzip
+
+# Install build dependencies from Intel repo.
+WORKDIR /root
+RUN curl -O https://download.01.org/intel-sgx/sgx-linux/2.24/distro/centos-stream9/sgx_rpm_local_repo.tgz && \
+tar -xaf sgx_rpm_local_repo.tgz && \
+microdnf -y install --nogpgcheck --repofrompath "sgx,file:///root/sgx_rpm_local_repo" libsgx-dcap-quote-verify-devel
+
+# Build.
+WORKDIR /usr/src/kbs
+COPY . .
+ARG KBS_FEATURES=coco-as-builtin,rustls,resource,opa
+RUN \
+cargo install --locked --root /usr/local/ --path kbs/src/kbs --no-default-features --features ${KBS_FEATURES} && \
+# Collect linked files necessary for the binary to run.
+mkdir -p /root/trustee/lib64 && \
+ldd /usr/local/bin/kbs | sed 's@.*\s/@/@' | sed 's/\s.*//' | xargs -I {} cp {} /root/trustee/lib64
```

Comment:
@spotlesstofu @bpradipt did you test this using TDX also? There are DCAP libs that do not show up using ldd.

Reply:
No, I didn't test with TDX. I can update the Dockerfile to keep all the contents of the RPMs, what do you think?

Reply:
The builder does not install the necessary runtime libraries. The final image is expected to have these: https://github.com/confidential-containers/trustee/blob/0b61fabcd3729bad00cb90a7d2eb5ea031eb4abc/kbs/docker/Dockerfile#L56-L57